websites/A/A3! Wiki/presence.ts

```diff
@@ -0,0 +1,60 @@
+const presence = new Presence({
+  clientId: "837305361639538728"
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
+let title;
+const actionURL = new URL(document.location.href);
+const title2URL = new URL(document.location.href);
```

Comment:
merge consts

Reply:
ah yea I saw the code checks from yesterday but I was waiting for your reply first, ok will work for it!